INSTALL.md

```diff
@@ -0,0 +1,326 @@
+CVC4 prerelease version 1.7.
+============================
```

Comment:
Suggestion: use `#` for consistency instead of `====`.

Reply:
See above, this is on purpose and not the same as #.